mjpc/trajectory.cc

```diff
@@ -108,6 +108,9 @@ void Trajectory::Rollout(
   times[0] = time;
   data->time = time;
 
+  // transition
+  task->Transition(model, data);
```

Comment:
These are needed for the keyframe transitions to function properly within the planner.